src/main/common/filter.h

```diff
@@ -33,6 +33,11 @@ typedef struct biquadFilter_s {
     float d1, d2;
 } biquadFilter_t;
 
+typedef union { 
+    biquadFilter_t biquad; 
+    pt1Filter_t pt1; 
+} filter_t;
```

Comment:
We should probably migrate other filters to use the same logic. Will save us some RAM

Reply:
Sure, as soon this is merged, i'm working on the accelerometer part, then will take a look at the other filters.